packages/services/f-feature-management/src/logger.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Default logger implementation.  The interface matches that used by various Fozzie components.
+ */
+const logger = {
+    logError (logMessage, store, logPayload = {}) {
+        if (console.error) {
```

Comment:
You might want to call this file `clientLogger` or something, and maybe checking that `console` also exists so it doesn't error if you try and access a property when console is undefined.

Reply:
I see the analytics below is also TODO so I'm assuming this is the same